src/DynamoCoreWpf/UI/GuidedTour/GuidesValidationMethods.cs

```diff
@@ -872,5 +882,31 @@ private static NodeView GetNotConnectedNode(Window stepMainWindow, string nodeTy
             return byOriginUserAdded;
         }
 
+
+        internal static void SubscribeRunButtonNextStep(Step stepInfo, StepUIAutomation uiAutomationData, bool enableFunction, GuideFlow currentFlow)
+        {
+            CurrentExecutingStep = stepInfo;
+            if (uiAutomationData.Parameters.Any() && uiAutomationData.Parameters[0] is string)
```

Comment:
I think you should assign the uiAutomationData.Parameters[0] to a variable so you can use it in several parts of the function, also you should include a validation for checking that Parameters is not null and the number of parameter are more than 1 or equals than 2.

Reply:
In this case, I'm expecting one parameter. I will assign it to a variable and check if it's null. Thanks!